packages/common-ts/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    "extends": "../../.eslintrc.js",
+    "rules": {
+        "prefer-arrow/prefer-arrow-functions": "off"
```

Comment:
Hmmm, why is this one different?

Reply:
So originally I had added package specific rules to make the linter happy, but completely removed that with the most recent commit. Now there are just one-off instances that the linter should ignore and all rules stem directly from the base `.eslintrc.js`